ethdk/src/Bls/BlsNetworks.ts

```diff
@@ -9,13 +10,13 @@ const localhost: BlsNetwork = {
   verificationGateway: '0x689A095B4507Bfa302eef8551F90fB322B3451c6',
 }
 
-const NETWORKS = {
-  localhost,
-}
-
-export function getNetwork(name?: string): BlsNetwork {
-  if (name === 'localhost' || name === undefined) {
-    return NETWORKS.localhost
+export function getNetwork(network?: Network): BlsNetwork {
+  if (network === undefined) {
```

Comment:
Open question, wouldn't this throw the "Unsupported network" error if null was passed in? If so, is that desired behaviour?

Reply:
Could remove the optional chaining operator on line 18 if we did decide to check null as well undefined on line 14